test/Router/IntegrationTest.php

```diff
@@ -353,21 +360,25 @@ public function testMatchWithAllHttpMethods(string $adapter, string $method): vo
     }
 
     /**
-     * @psalm-return array<string, array{
+     * @psalm-return iterable<array{
      *     0: class-string,
      *     1: RequestMethod::METHOD_*
      * }>
      */
```

Comment:
I'd expect us to be more specific here too

```suggestion
     * @psalm-return Generator<int, array{class-string<RouterInterface>, RequestMethod::METHOD_*}>
     */
```

Reply:
I added the interface to the annotation 👍🏼 

I had to retain the array index in the annotation though, otherwise Psalm would error.